### PR TITLE
WRN-1345: Migrate BodyText and Heading unit tests to Testing Library

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated `BodyText` and `Heading` unit tests to react-testing-library

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1345

### CODE REVIEW

File: BodyText/tests/BodyText-specs.js [hunk 1]
```diff
@@ -1,39 +1,51 @@
...
+		const {getByTestId} = render(<BodyText data-testid="bodyText" noWrap />);
+		const bodyText = getByTestId('bodyText');
+		const marquee = bodyText.children.item(0);
 
-		const expected = true;
-		const actual = subject.findWhere(c => c.name() === 'ui:Marquee').exists();
+		const expected = 'marquee';
 
-		expect(actual).toBe(expected);
+		expect(marquee.className).toBe(expected);
```
Comment (alexandrumorariu) [inline BodyText/tests/BodyText-specs.js:22]: Can you place marquee.className in an const actual, we should keep the actual => expected notation
(same for the ones below)

**Review by alexandrumorariu — DISMISSED** [review]

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: BodyText/tests/BodyText-specs.js [hunk 2]
```diff
@@ -1,38 +1,54 @@
...
+
+		expect(actual).toContain(expected);
+	});
+
+	test('should have small class if `size` is true', () => {
+		const {getByTestId} = render(<BodyText data-testid="bodyText" size="small" />);
+		const bodyText = getByTestId('bodyText');
+
+		const expected = 'small';
+		const actual = bodyText.className
```
Comment (alexandrumorariu) [inline BodyText/tests/BodyText-specs.js:41]: ; here

File: Heading/tests/Heading-specs.js [hunk 3]
```diff
@@ -1,33 +1,50 @@
...
+	test('should add the showLine class', () => {
+		const {getByTestId} = render(<HeadingBase data-testid="heading" showLine>check</HeadingBase>);
+		const heading = getByTestId('heading');
 
-		const expected = content;
-		const actual = subject.text();
+		const expected = 'showLine';
 
-		expect(actual).toBe(expected);
+		expect(heading).toHaveClass(expected);
```
Comment (stanca-pop-lgp) [inline Heading/tests/Heading-specs.js:21]:
```suggestion
		expect(heading.className).toContain(expected);
```

Comment (stanca-pop-lgp) [reply Heading/tests/Heading-specs.js:21]: You have to replace that line in all following tests otherwise the tests will fail.

Comment (adrian-cocoara-lgp) [reply Heading/tests/Heading-specs.js:21]: I forgot to import '@testing-library/jest-dom'. Should be ok now

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**